basicswap/static/js/bids_sentreceived.js

```diff
@@ -684,27 +847,27 @@ const createTableRow = async (bid) => {
                 </div>
             </td>
 
-            <!-- Status Column -->
-            <td class="p-3">
-                <div class="flex justify-center" data-tooltip-target="tooltip-status-${uniqueId}">
-                    <span class="w-full xl:w-4/5 flex bold justify-center items-center px-2.5 py-1 rounded-full text-xs font-medium
-                                ${getStatusClass(bid.bid_state)}">
-                        ${bid.bid_state}
-                    </span>
-                </div>
+           <!-- Status Column -->
+           <td class="py-3 px-6">
+            <div class="relative flex justify-center" data-tooltip-target="tooltip-status-${uniqueId}">
+                <span class="px-2.5 py-1 inline-flex items-center rounded-full text-xs font-medium ${getStatusClass(bid.bid_state)}">
```

Comment:
```suggestion
           <td class="py-3 px-6">
            <div class="relative flex justify-center" data-tooltip-target="tooltip-status-${uniqueId}">
                <span class="w-full lg:w-7/8 xl:w-2/3 px-2.5 py-1 inline-flex items-center justify-center rounded-full text-xs font-medium bold ${getStatusClass(bid.bid_state)}">
```

i made the previous change to made the status icons a uniform width, but they were a bit too big on larger resolutions.

i think this should work better